### DISCUSSION TITLE
RAM/CPU requirements

### DISCUSSION THREAD

**Discussion by dekiesel** [TandoorRecipes/recipes#2103] [Q&A] [answered]
Hi,

I have a very old NAS (Zyxel NAS 310, 256MB RAM, 1,2GHz Single Core) and was wondering whether I could run recipes on that NAS.

I couldn't find minimal requirements in the docs, so if somebody could let me know how likely this would work I'd be grateful.

Thanks!

**Comment by vabene1111** [comment, marked as answer]
generally speaking tandoor does not need much, i recommend just giving it a try, i run it on a single core VPS with 1 GB of RAM and 5GB of storage and many people are using rapberry pis so your NAS should work

**Reply by dekiesel** [reply]
Thank you for the answer!
